snowflake-table-grant-all/module_test.go

```diff
@@ -0,0 +1,40 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/chanzuckerberg/go-misc/tftest"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestModule(t *testing.T) {
+	test := tftest.Test{
+		// just run init, swtich to Plan or Apply when you can
+		Mode: tftest.Init,
+
+		Setup: func(t *testing.T) *terraform.Options {
+			vars := map[string]interface{}{
+				"database_name": "database",
+				"schema_name":   "schema",
+				"table_name":    "table",
+				"roles":         []string{"a", "b", "c"},
+				"shares":        []string{"as", "bs", "cs"},
+			}
+
+			opts := &terraform.Options{
```

Comment:
Use [tftest.Options](https://github.com/chanzuckerberg/go-misc/blob/main/tftest/util.go#L152)

Reply:
https://github.com/chanzuckerberg/go-misc/blob/main/tftest/util.go#L155-L159 breaks things for me by including the extra vars

Reply:
oh derp